examples/sampling/graphbolt/rgcn/hetero_rgcn.py

```diff
@@ -578,7 +578,7 @@ def main(args):
     # `institution` are generated in advance and stored in the feature store.
     # For `ogbn-mag`, we generate the features on the fly.
     embed_layer = None
-    if args.dataset == "ogbn-mag":
+    if args.dataset == "ogbn-mag-seeds":
```

Comment:
This looks weird to me. User should not care about `seeds` suffix at all. the dataset name exposed to user should always be `ogbn-mag`. In the near future, we will fully deprecate existing`BuiltinDataset(ogbn-mag)`, just replace it with `ogbn-mag-seeds`.

Reply:
and README is not updated accordingly.